src/browser/connection/index.ts

```diff
@@ -40,6 +40,10 @@ interface InitScriptTask extends InitScript {
     resolve: Function;
 }
 
+interface ProviderMetaInfoOptions {
+    changePrettyUserAgent?: boolean;
```

Comment:
`appendToUserAgent` could be a better name, I think

Reply:
I propose the `canBeAppendedToPrettyUserAgent` name.

Reply:
The way I see it, `canBeAppendedToPrettyUserAgent` is a bit overcomplicated and leads to leaky abstraction, because browser provider developer should not know anything about internal form of UA representation (which `prettyUserAgent` is).